### DISCUSSION TITLE
JSON Files

### DISCUSSION THREAD

**Discussion by reujoal** [JasonLautzenheiser/trizbort#559] [Issues / Bugs]
I have someone helping me that uses the Trizbort.io browser version. Problem is, it saves the files as JSON files, which I cannot open in Trizbort 1.7.4.0 
Is there any way to import them or change the file format? I understand that TRIZBORT files can be opened in the browser version, but not visa versa. Please help <3

**Comment by JasonLautzenheiser** [comment]
Trizbort.io is developed by someone else and unfortunately doesn't appear like it's been updated in over a year, so not sure if it's still being maintained.   Having said that, I will look at adding a feature to this desktop version to import the json files that the web version creates.

Unfortunately, currently I don't really have a solution for you.
